lib/mpl_toolkits/axisartist/grid_helper_curvelinear.py

```diff
@@ -339,8 +339,9 @@ def __init__(self, aux_trans,
         (inverse transform should be defined) or a tuple of two callable
         objects which defines the transform and its inverse. The callables
         need take two arguments of array of source coordinates and
-        should return two target coordinates:
-          e.g., x2, y2 = trans(x1, y1)
+        should return two target coordinates.::
+
+            e.g., x2, y2 = trans(x1, y1)
```

Comment:
gah crap you're right - I'll just make it "e.g., ``x2, y2 = trans(x1, y1)``" sounds good?